wayland-client/src/conn.rs

```diff
@@ -271,6 +272,13 @@ impl<'a> ConnectionHandle<'a> {
     }
 }
 
+impl<'a> From<&'a mut Handle> for ConnectionHandle<'a> {
+    /// Creates a [`ConnectionHandle`] using a [`&mut Handle`](Handle) from `wayland-backend`.
+    fn from(handle: &'a mut Handle) -> Self {
+        ConnectionHandle { inner: HandleInner::Handle(handle) }
+    }
```

Comment:
Such trait impls are not very discoverable in rustdoc output, I think it'd be helpful to mention it in the type-level documentation.